apps/wakunode2/app.nim

```diff
@@ -596,13 +596,34 @@ proc startRestServer(app: App, address: ValidIpAddress, port: Port, conf: WakuNo
     rest_legacy_filter_api.installLegacyFilterRestApiHandlers(server.router, app.node, legacyFilterCache)
 
     let filterCache = rest_filter_api.MessageCache.init()
-    rest_filter_api.installFilterRestApiHandlers(server.router, app.node, filterCache)
+
+    let filterDiscoHandler = 
+      if app.wakuDiscv5.isSome():
+        some(defaultDiscoveryHandler(app.wakuDiscv5.get(), Filter))
+      else: none(DiscoveryHandler)
```

Comment:
> Instead of returning an empty (none) discovery handler, should we return a default discovery service?
> 
> Maybe we could define a "staticDiscovery" handler that could return a hard-coded peers. I suggest then to not use an `Option[...]` type and always return a valid discoveryHandler. That way, the code in `rest/*/handlers.nim` will get simpler.
> 
> If this is not possible, I think we should end the execution at this point and indicate that a proper disco handler should be set. Better to do that than later on responding to the REST clients that "there is no disc service configured" or the current message that is being returned in the next snippet:
> 
> ```
>     let handler = discHandler.valueOr:
>       return makeRestResponse(
>         req.requestId,
>         FilterSubscribeError.serviceUnavailable("No suitable peers")
> ```

Users can already specify service peer if they want to.